Makefile

```diff
@@ -523,9 +529,83 @@ check-jsonnet-tests: build-jsonnet-tests
 check-tsdb-blocks-storage-s3-docker-compose-yaml:
 	cd development/tsdb-blocks-storage-s3 && make check
 
+# Generate packages for a Mimir release.
+FPM_OPTS := fpm -s dir -v $(VERSION) -n mimir -f \
+	--license "AGPL 3.0" \
+	--url "https://github.com/grafana/mimir" \
```

Comment:
What does this URL refer to? In service files, we use `https://grafana.com/oss/mimir/` instead.

Reply:
This URL ends up in the documentation link of the service file
```
root@f4d35967e37d:/mnt# cat /lib/systemd/system/mimir.service
# SPDX-License-Identifier: AGPL-3.0-only
# Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/packaging/deb/systemd/cortex.service
# Provenance-includes-license: Apache-2.0
# Provenance-includes-copyright: The Cortex Authors.

[Unit]
Description=Horizontally scalable, highly available, multi-tenant, long term Prometheus.
Documentation=https://grafana.com/oss/mimir/
Wants=network-online.target
After=network-online.target

[Service]
Restart=always
User=mimir
EnvironmentFile=/etc/default/mimir
ExecStart=/usr/local/bin/mimir --log.level $LOG_LEVEL $CUSTOM_ARGS
ExecReload=/bin/kill -HUP $MAINPID
TimeoutStopSec=20s
SendSIGKILL=no
WorkingDirectory=/var/lib/mimir

[Install]
WantedBy=multi-user.target
```